### DISCUSSION TITLE
using STARTTLS with SendMail

### DISCUSSION THREAD

**Discussion by checpg** [cybertec-postgresql/pg_timetable#644] [Q&A]
Hello dear community,
is there a possibility to use the SendMail functionality of pg_timetable with STARTTLS and use certificates for authentication?

**Comment by pashagolub** [comment]
Hi. I'm not sure it's implemented right now but I believe `SendMail` can be extended for that.

**Reply by checpg** [reply]
Hello, is it possible to place this as a feature request for pg_timetable? (adding STARTTLS to SendMail and the use of certificates for authentication)

**Reply by pashagolub** [reply]
Sure! Sounds OK to me. Do you want to try to implement that? :)

**Reply by checpg** [reply]
That would be great and a nice feature to have. Many thanks!
